### PR TITLE
Spraypaint BigHacks: Adding Copyright Information to Footer

### PR DESCRIPTION
**Pull request by bc-johnhebron**
Adding in copyright information into the footer at parity to Blueprint style themes.
In the format of (c) current_year store_name.

### CODE REVIEW

**Comment by bc-johnhebron** [comment]
@bc-julianleong @bc-adamferenzi @cristycarpenter

File: templates/components/common/footer.html [hunk 1]
```diff
@@ -66,5 +66,13 @@ <h5 class="footer-info-heading">{{lang 'footer.info'}}</h5>
                 <p class="powered-by">Powered by <a href="http://bigcommerce.com" rel="nofollow">BigCommerce</a></p>
             </div>
         {{/if}}
+        <div class="footer-copyright">
+            <p class="powered-by">&copy; <span id="copyright_year"></span> {{settings.store_name}} </p>
+        </div>
+        <script type="text/javascript">
+            var d = new Date();
+            var n = d.getFullYear();
+            document.getElementById("copyright_year").innerHTML = n;
```
Comment (mcampa) [inline templates/components/common/footer.html]: I would rewrite this to one line:

```js
document.getElementById("copyright_year").innerHTML = new Date().getFullYear();
```

**Comment by bc-julianleong** [comment]
Will this include a control in theme editor in case a merchant doesn't want to display this or if they want to use something else besides the store name for the copyright?

**Comment by mcampa** [comment]
Agree with @bc-julianleong, the html should be wrapped by a toggle in theme-editor

**Comment by bc-adamferenzi** [comment]
I think the latter suggestion of custom store name would need to be a phase three feature with text boxes.  Adding an on/off toggle would probably be pretty straightforward.

**Comment by bc-julianleong** [comment]
Hey Adam - could you add the on/off toggle for this feature to the Footer accordion in Theme Editor and then we can merge it.

**Comment by bc-adamferenzi** [comment]
This pull is @bc-johnhebron so I'll leave it with him, but yes, shouldn't be a problem.  Thanks!

**Comment by bc-johnhebron** [comment]
I just pushed the new changes with the toggle @bc-julianleong 
CC: @bc-adamferenzi

**Comment by mcampa** [comment]
LGTM 👍

**Comment by bc-johnhebron** [comment]
Do I just click "Close" or do you need to accept the pull request on your side? It's my first PR :)

**Comment by mcampa** [comment]
You need to click on "Merge pull request". But I think you don't have permissions for this repo. I'll do it for you

**Pull request merged**